src/models/User.js

```diff
@@ -46,13 +46,17 @@ class User extends Resource {
   }
 
   /**
+   * @param {object} queryParameters
    * @returns {Promise<User>}
    */
-  update() {
-    return this.client.updateUser(this.id, this);
+  update(queryParameters) {
+    return this.client.updateUser(this.id, this, queryParameters);
```

Comment:
Since these seem breaking changes, what's the way to avoid failures here? Can users omit these additional parameters?

Reply:
Params can be omitted, no breaking change here